src/content/pages/faq.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: FAQ
+subtitle: Frequently Asked Questions about EuroPython
+---
+
+# FAQ
+
+## Milestones
+
+<ul  className="milestone-done">
+  <li>**12 December 2024 Call for Teams starts</li>
```

Comment:
```suggestion
  <li>**12 December 2024** [Call for Contributors](https://forms.gle/kY7jqSJTjEhFccWJ6) starts</li>
```
That's the name on the form.

Reply:
I don't want to link to the form as the form is still reachable but the Call is through for now.

Reply:
Ah OK! Understood.